src/nvim/getchar.c

```diff
@@ -1847,7 +1849,7 @@ static mapblock_T *find_typed_map(const bool timedout, const int local_State,
     return NULL;
   }
 
-  int keylen = *keylenp;
+  int keylen = 0;
```

Comment:
Seems like keylen is only used much later in the code. Declaration could be moved.